src/app/services/http-services/Membership.js

```diff
@@ -0,0 +1,66 @@
+angular.module('app')
+  .factory('Membership', ($q, $http) => {
+    let baseUrl = 'http://localhost:3000/api/membership'
```

Comment:
Oooohh Good thought. Making these work a little more dynamically will be an interesting little challenge. It's not too hard, but it will take some thought and careful planning. It will have to be done through environment variables.

I will look into making it work.